commit ea9bffdc48df6693fe996387b4f54e983e88f6d2 (from c8d0ca9337211e6cd36f448df28f4f99e5f82591)
Merge: c8d0ca9 0de8261
Author: nothinman <martin@lukasik.name>
Date:   Mon Jun 2 18:02:36 2014 +0100

    Merge pull request #949 from foosel/fix/M30_clearToSend_bug
    
    M30 response is missing linefeed, "ok" therefore not on own line

diff --git a/Marlin/cardreader.cpp b/Marlin/cardreader.cpp
index 5fb8dcc..d2fb418 100644
--- a/Marlin/cardreader.cpp
+++ b/Marlin/cardreader.cpp
@@ -437,7 +437,7 @@ void CardReader::removeFile(char* name)
     if (file.remove(curDir, fname)) 
     {
       SERIAL_PROTOCOLPGM("File deleted:");
-      SERIAL_PROTOCOL(fname);
+      SERIAL_PROTOCOLLN(fname);
       sdpos = 0;
     }
     else

